### DISCUSSION TITLE
Example for boxes in no gravity.

### DISCUSSION THREAD

**Discussion by jvink** [pmndrs/use-cannon#80] [General] [answered]
Is there an example for useBox's to interact with each other in 0 G. I know you can set the gravity to [0,0,0] on the Physics component. But the boxes, should then move to each other.

Code:
https://codesandbox.io/s/r3f-cannon-instanced-physics-siqiq

**Comment by stockhuman** [comment, marked as answer]
I don't think this library can provide theoretically-accurate physics for you; bodies do not simulate their own gravitational attraction, unfortunately. You can, however, apply a force to every body to simulate this on every frame.

**Reply by jvink** [reply]
Good idea! I'll give it a try.

**Comment by codynova** [comment]
There was also some discussion about an `<Attractor />` component or something similar here #67. The solution posted here may work in your case https://github.com/react-spring/use-cannon/issues/67#issuecomment-619453231

I'm also interested in updating the API to support world event calls, but have to find the time.
